keras_hub/src/models/yolo_v8/ciou_loss.py

```diff
@@ -0,0 +1,86 @@
+import keras
+from keras import ops
+
+from keras_hub.src.bounding_box.iou import compute_ciou
+
+
+class CIoULoss(keras.losses.Loss):
+    """Implements the Complete IoU (CIoU) Loss
+
+    CIoU loss is an extension of GIoU loss, which further improves the IoU
+    optimization for object detection. CIoU loss not only penalizes the
+    bounding box coordinates but also considers the aspect ratio and center
+    distance of the boxes. The length of the last dimension should be 4 to
+    represent the bounding boxes.
+
+    Args:
+        bounding_box_format: a case-insensitive string (for example, "xyxy").
+            Each bounding box is defined by these 4 values. For detailed
+            information on the supported formats, see the [KerasCV bounding box
+            documentation](https://keras.io/api/keras_cv/bounding_box/formats/).
```

Comment:
add link to keras instead

Reply:
Hi @divyashreepathihalli I was unable to find in the keras documentation the available box formats. I am linking to the source keras source code. We can update this once the documentation is available.